python/test/regression/test_performance.py

```diff
@@ -148,13 +148,13 @@ def test_elementwise(N, dtype_str):
         (4, 48, 1024, 16, True, True, 'forward', 'float32'): 0.155,
         (4, 48, 4096, 64, True, True, 'backward', 'float16'): 0.203,
         (4, 48, 4096, 64, True, True, 'backward', 'bfloat16'): 0.202,
-        (4, 48, 1024, 16, True, True, 'backward', 'float32'): 0.108,
```

Comment:
It looks like these numbers are in ms, so doesn't this change mean that the performance got *worse*?

Reply:
It's a ratio of peak performance.

Reply:
Ah, I see that now.  Thanks!